partials/language-specific-deploy/php.md

```diff
@@ -480,7 +494,9 @@ Then, set `APP_LOG=syslog` as Clever application environment variable.
 
 ## Using HTTP authentication
 
-Using basic HTTP authentication, PHP usually gives you the user and password in variables named `$_SERVER['PHP_AUTH_USER']` and `$_SERVER['PHP_AUTH_PW']`. On Clever Cloud, we have enabled an option in Apache to pass directly the Authorization header even though we are using FastCGI; still, PHP does not use the header and the variables mentioned before are empty.
+Using basic HTTP authentication, PHP usually gives you the user and password in variables named `$_SERVER['PHP_AUTH_USER']` and `$_SERVER['PHP_AUTH_PW']`.
+
+On Clever Cloud, we have enabled an option in Apache to pass directly the Authorization header even though we are using FastCGI; still, PHP does not use the header and the variables mentioned before are empty.
```

Comment:
```suggestion
At Clever Cloud, we have enabled an Apache option to pass directly the Authorization header, even though we are using FastCGI; still, the header is not used by PHP, and the aforementioned variables are empty.
```

Because this variant: 

> PHP does not use the header and the variables…

implies for a split  second of the reader's attention that both header and variables are objects sharing the same fate of not being used by PHP. 
Using the passive voice solves the issue.